service/trino-verifier/src/main/java/io/trino/verifier/Verifier.java

```diff
@@ -243,9 +243,6 @@ private VerifierQueryEvent buildEvent(Validator validator)
             }
         }
 
-        Stream<QueryResult> controlQueries = concat(Stream.of(control), validator.getControlPreQueryResults().stream(), validator.getControlPostQueryResults().stream());
-        Stream<QueryResult> testQueries = concat(Stream.of(test), validator.getTestPreQueryResults().stream(), validator.getTestPostQueryResults().stream());
```

Comment:
what does it change?

Reply:
Turns out the stream created by the `Stream.concat` cannot be iterated more than once.

Reply:
Oh - yeah. I think this is generally true for streams.